### ISSUE TITLE
Generate QC reports

### ISSUE THREAD

**Issue by glatard** [SIMEXP/cbrain-plugins-psom#11]
CBRAIN has a framework to display QC panels specific to a particular file type, and to tag these files accordingly. A QC framework can be implemented by:
- [x] creating a specific type of PSOM or Niak results. Done. All files ending with ".niak" will be associated with a particular CBRAIN file type that will have the Niak viewer and QC panel.
- [ ] The viewer and QC panels associated to this file type will show HTML pages produced by Niak. These webpages will be generated by Niak once https://github.com/SIMEXP/niak/issues/176 is implemented.

@pbellec also suggested that:
- [x] QC tags are saved in the data. At the moment this cannot be done easily but a CSV file containing all the QC tags can be exported from the file list. 
- [x] Comments can be added to QC tags. Available by default in CBRAIN.
- [ ] Viewer is also used while pipeline is running, to monitor progression.

**Comment by glatard** [comment]
On hold, waiting for a release of Niak that would produce HTML pages (has to be produced by @poquirion and/or @pbellec).

**Comment by lunebellec** [comment]
I am integrating the generation of the report for fMRI preprocessing in the preprocessing pipeline itself. Relevant branch in niak is here:
https://github.com/SIMEXP/niak/tree/issue202

associated issue is there:
https://github.com/SIMEXP/niak/issues/202

Planning on wrapping that up this week.

**Comment by lunebellec** [comment]
the reports are available in the master brain. Waiting on @poquirion to release niak-cog-beta
